spec/dummy/config/database.yml

```diff
@@ -1,15 +1,14 @@
 defaults: &defaults
-  host: localhost
+  host: <%= ENV.fetch('THREDDED_DB_1_PORT_5432_TCP_ADDR', 'localhost') %>
```

Comment:
This will definitely work as-is. It might still be helpful to know that the container *link* you have defined in `docker-compose.yml` above will also [create a fake hostname](https://docs.docker.com/userguide/dockerlinks/#updating-the-etchosts-file) that refers to the linked container. Within your `web` container, any running applications can connect to a hostname named `DB` which will point at the defined `db` container's IP. That feature is kind of awkward to summarize, so please let me know it's still unclear.